packages/dev/loaders/src/glTF/2.0/glTFLoader.ts

```diff
@@ -1083,15 +1083,10 @@ export class GLTFLoader implements IGLTFLoader {
                                         divider = 65535.0;
                                         break;
                                 }
-                                for (let i = 0; i < 3; ++i) {
-                                    mmin[i] = Math.max(mmin[i] / divider, -1.0);
-                                    mmax[i] = Math.max(mmax[i] / divider, -1.0);

```

Comment:
The main issue is these two lines. This actually changes the values of the glTF accessors which can be processed multiple times, in which case it gets smaller each time.